examples/expr2.py

```diff
@@ -1,10 +1,6 @@
 def main0():
-    x: i32
-    x = (2+3)*5
+    i: list[i32] = [1, 2, 3]
+    j: list[i32] = [4, 5, 6]
+    x: list[i32] = j + i
     print(x)
```

Comment:
This example fail in the llvm with `AssertFailed: std::find(nested_globals.begin(), nested_globals.end(), x_h) != nested_globals.end()`